string name
string jointName
---
float64 posX
float64 posY
float64 posZ
